kontonummer.js

```diff
@@ -425,4 +484,27 @@
         }
         return sum && sum % 11 === 0;
     };
-}(window));
+
+    /**
+     * Fyll nollor för Swedbank bankkontonumren
+     *
+     * @param {String} number Bankkontonummer
+     * @param {Object} bank Bank object
+     * @returns {String} Bankkontonummer med nollor
+     */
+    var fillZeros = function (accountNumber, bank) {
+        var numberWithoutClearing = accountNumber.substr(bank.lengths.clearing);
```

Comment:
How about creating a function for this? It could be reused for the return value if we still want to keep returning `account_number` and `clearing_number`.